frontend/www/js/omegaup/components/common/Typeahead.vue

```diff
@@ -32,12 +32,15 @@ import { types } from '../../api_types';
 })
 export default class Typeahead extends Vue {
   @Prop() existingOptions!: types.ListItem[];
+  @Prop({ default: () => [] }) currentOptions!: types.ListItem[];
```

Comment:
mismo comentario que en el otro PR: el prefijo `current` es nada más para las cosas que se modifican adentro del componente.

```suggestion
  @Prop({ default: () => [] }) options!: types.ListItem[];
```

Reply:
Listo